UserApi/UserApi/Controllers/UserController.cs

```diff
@@ -39,6 +39,7 @@ public UserController(IUserAccountService userAccountService, TelemetryClient te
         [SwaggerOperation(OperationId = "CreateUser")]
         [ProducesResponseType(typeof(NewUserResponse), (int)HttpStatusCode.Created)]
         [ProducesResponseType((int)HttpStatusCode.NotFound)]
+        [ProducesResponseType((int)HttpStatusCode.Conflict)]
```

Comment:
remember to put in a `typeof(UserCreationConflictResponse)` here to hint of the response type